W-12204799-Add-Kubernetes-service-note-CP (#448)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc

      y-test: false
    host: test.com
----
+

> . Enter the following command to view the pods logs and ensure that the API is created successfully: +

Reviewer: ```suggestion
. Enter the following command to view the pods logs and ensure that the API was created successfully: +
```

In my head, this should not be changed. The API was already created when you're looking at this. I think you would only use "is" if you're monitoring the process of it being created.

Author: OK! Thanks!

----
kubectl logs -f <pod-name>
----

> If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream or you make a request from a policy, you must set:

Reviewer: ```suggestion
If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, you must set the following parameters in the `service` resource:
```

Author: OK! Thanks!

> * `spec.ports[].protocol`, which must be TCP

Reviewer: ```suggestion
* `spec.ports[].protocol` to `TCP`
```

Author: OK! Thanks!

* `spec.ports[].name`

> Your `spec.ports[].name` service must adhere to the `<protocol>[-<suffix>]` structure. `[-<suffix>]` is optional, and the valid options for `<protocol>` are:

Reviewer: Is suffix a place holder?

If so, what happens if you don't include it? What protocol is used?

Reviewer: Also, is this bullet list related to the above bullet list?

Author: Yes, the suffix is a placeholder and, as it is optional, it's fine if you don't include it.
Regarding the list, it is part of the same topic -sort of.
Does it make sense, @glenn-rodgers-sf?
Thanks!
